composer.json

```diff
@@ -11,7 +11,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.2",
-        "symfony/event-dispatcher": "^2.0 || ^3.0"
+        "symfony/event-dispatcher": "^2.3 || ^3.0"
```

Comment:
@SpacePossum: FWIW, the master branch's test suite still works against Symfony 2.0.4. I changed the lower requirement to "^2.0" and ran the following:

```
$ composer update --prefer-lowest
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Removing symfony/event-dispatcher (2.2.x-dev)
  - Installing symfony/event-dispatcher (2.0.4)
    Downloading: 100%         

Writing lock file
Generating autoload files

$ phpunit 
PHPUnit 4.8.26 by Sebastian Bergmann and contributors.

.......................S..

Time: 127 ms, Memory: 4.00MB

OK, but incomplete, skipped, or risky tests!
Tests: 26, Assertions: 105, Skipped: 1.
```

> Would you be open to PR with bumping versions of active security support for reasons I've talked about?

@TomasVotruba: I'm not open to modifying this library's dependencies for that purpose. Folks can use a project such as [roave/security-advisories](https://github.com/Roave/SecurityAdvisories), which is regularly updated, to catch insecure versions of packages.